pkg/gardenlet/controller/shoot/shoot_control.go

```diff
@@ -505,12 +502,13 @@ func (c *Controller) durationUntilNextShootSync(shoot *gardencorev1beta1.Shoot)
 }
 
 func (c *Controller) updateShootStatusAndRequeueOnSyncError(g gardencore.Interface, shoot *gardencorev1beta1.Shoot, err error) (reconcile.Result, error) {
-	message := fmt.Sprintf("Shoot cannot be synced with Seed: %v", err)
-	c.recorder.Event(shoot, corev1.EventTypeNormal, gardencorev1beta1.EventOperationPending, message)
+	msg := fmt.Sprintf("Shoot cannot be synced with Seed: %v", err)
+	shootLogger := logger.NewShootLogger(logger.Logger, shoot.ObjectMeta.Name, shoot.ObjectMeta.Namespace)
```

Comment:
Can you pass the logger from the calling code to this function instead of creating a new one?